MH4U Dex/AreaDropTableViewCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AreaDropTableViewCell.h
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/11/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import "AreaDrop.h"
```

Comment:
Depending on how it's used in the implementation file, it may or may not be necessary to add `#import "AreaDrop.h"` in `AreaDropTableViewCell.m`—Xcode generally gives an error talking about forward-declarations if you try to do something with the class that requires the import.